docs/content/2.deploy/providers/cloudflare.md

```diff
@@ -12,6 +12,10 @@ description: 'Discover Cloudflare preset for Nitro!'
 **Note:** This preset uses [service-worker syntax](https://developers.cloudflare.com/workers/learning/service-worker/) for deployment.
 ::
 
+::alert{type="warning"}
+**Warning:** Please be aware that `runtimeConfig` doesn't work with the Cloudflare preset (see [#272](https://github.com/unjs/nitro/issues/272) for more). As a workaround, you can use Cloudflare env variables as constants in the code.
```

Comment:
Probably we can change the note that runtime config cannot be updating using environment variables with Cloudflare preset?

Reply:
Done ☺️